src/ui/page/share/dataset/steps/summary/summary.tsx

```diff
@@ -94,6 +112,27 @@ export const Summary: FC = () => {
 
         return null
       }
+
+      case 'date-range': {
+        const dateRangeValue = getDateRangeFieldValue(item)
+
+        if (!dateRangeValue || (!dateRangeValue.from && !dateRangeValue.to)) return null
+
+        const { from, to } = dateRangeValue
+
+        return from && to ? (
+          <p className="okp4-dataverse-portal-share-dataset-summary-item-text">
+            {`${t('from')} ${formatISODate(from)} ${t('to').toLowerCase()} ${formatISODate(to)}`}
+          </p>
+        ) : (
+          <p className="okp4-dataverse-portal-share-dataset-summary-item-text">
+            {from
+              ? `${t('from')} ${formatISODate(from)}`
+              : `${t('to')} ${formatISODate(to as string)}`}
+          </p>
+        )
+      }
```

Comment:
A bit tricky for me to read those nested ternaries and interpolations in JSX part.
 
Could you please make it more readable 🙏?

Reply:
For sure! It has been rewrite in a more verbose, but easier to read way.